contracts/core/Allo.sol

```diff
@@ -53,30 +70,32 @@ contract Allo is Transfer, Initializable, Ownable, AccessControl {
     /// @notice Incremental index
     uint256 private _poolIndex;
 
-    /// @notice msg.sender -> nonce for cloning strategies
-    mapping(address => uint256) private _nonces;
-
     /// @notice Allo treasury
     address payable public treasury;
 
     /// @notice Registry of pool creators
     Registry public registry;
 
+    /// @notice msg.sender -> nonce for cloning strategies
+    mapping(address => uint256) private _nonces;
+
     /// @notice Pool.id -> Pool
     mapping(uint256 => Pool) public pools;
 
     /// @notice Strategy -> bool
     mapping(address => bool) public approvedStrategies;
 
+    /// @notice Reward for catching fee skirting (1e18 = 100%)
+    uint256 public feeSkirtingBounty;
```

Comment:
rename this to `feeSkirtingBountyPercentage` ?

Reply:
Done.